app/main.py

```diff
@@ -16,6 +16,7 @@
 # Get settings from Environment with Pydantic BaseSettings
 class Settings(BaseSettings):
     teraslice_url: str = "http://localhost:5678"
+    cacert_file: str | None = None
```

Comment:
Technically this is a file path, is `str` the best choice for the type or is there an actual filepath type?